.docker/app/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:11.10.0-alpine as prod
```

Comment:
IMHO you can remove the target name as there is only one stage in this dockerfile.
Also, what about using a more recent version of node ?